js/adapt-contrib-pageLevelProgress.js

```diff
@@ -9,7 +9,12 @@ define([
     var PageLevelProgress = Backbone.Controller.extend({
 
         initialize: function() {
-            this.listenTo(Adapt, 'app:dataReady', this.onDataReady);
+            Adapt.on({
+                'app:dataReady': this.onDataReady.bind(this),
+                'app:languageChanged': function() {
+                    this.stopListening();
```

Comment:
Can you add a comment about this call removing on the events created by `setUpEventListeners`?
```js
                 'app:languageChanged': function() {
                    // Remove events created by setUpEventListeners
                    this.stopListening();
```

Reply:
might also be worth referencing [this issue](https://github.com/jashkenas/backbone/issues/4210) as to why the syntax is a little odd

Reply:
👍